subsys/bluetooth/audio/bap_broadcast_source.c

```diff
@@ -870,6 +943,23 @@ int bt_bap_broadcast_source_reconfig(struct bt_bap_broadcast_source *source,
 
 			stream_param = &subgroup_param->params[j];
 			stream = stream_param->stream;
+			#if CONFIG_BT_AUDIO_CODEC_CFG_MAX_DATA_SIZE > 0
```

Comment:
```suggestion
			if (CONFIG_BT_AUDIO_CODEC_CFG_MAX_DATA_SIZE > 0)
```

All the data inside of this seems to be available even when `CONFIG_BT_AUDIO_CODEC_CFG_MAX_DATA_SIZE` so we can just use a regular `if`, right?